### PR TITLE
Cope with "generic scalar" types, instead of crashing.

### PR DESCRIPTION
**Pull request by tonyhignett**

### CODE REVIEW

File: dist/module.js [hunk 1]
```diff
@@ -21140,6 +21140,7 @@ function (_super) {
 
           var dataFrameArray = [];
           var queryType = results.pop();
+          console.log("hello!");
```
Comment (JonathanWylie) [inline dist/module.js]: Debug log

File: dist/module.js [hunk 2]
```diff
@@ -21278,9 +21279,11 @@ function (_super) {
                           };
                         }
 
+                        console.log("df", dataFrame);
```
Comment (JonathanWylie) [inline dist/module.js]: Debug log?

File: dist/module.js [hunk 3]
```diff
@@ -21278,9 +21279,11 @@ function (_super) {
...
                         dataFrameMap.set(identifiersString, dataFrame);
                       }
 
+                      console.log("doc", doc);
```
Comment (JonathanWylie) [inline dist/module.js]: Debug log?

**Review by JonathanWylie — CHANGES_REQUESTED** [review]
I don't need to see this again.

**Comment by tonyhignett** [comment]
I'd deleted these from the code itself but I must not have rebuilt so they're still in the built version. I rebuilt and they're gone.

**Pull request merged**